shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx

```diff
@@ -12,27 +12,12 @@ keywords:
   - explanation
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 import WarningUnlockConditions from '../_admonitions/_warning-unlock-conditions.md';
 
 # Stardust Exchange Integration Guide
 
-:::note
-
-The Stardust update is a set of protocol changes that were first deployed to the Shimmer network.
-See [What is Stardust?
-](https://wiki.iota.org/shimmer/develop/explanations/what-is-stardust/) for an overview of its features or [TIPs](https://github.com/iotaledger/tips) flagged `Stardust` for a deeper understanding of the protocol changes.
-
-:::
-
-:::warning
-
-The Stardust update allows for complex tokenization schemes that can lead to monetary losses if misused.
-Transaction outputs might have multiple unlocking conditions. These conditions may require the return of part or all of the output amounts, have an expiration date if not claimed on time, or they may not be unlockable for an extended period of time. Therefore, you need to review these unlock conditions carefully before accepting any deposits to avoid loss of funds.
-
-:::
```

Comment:
Is there a reason why this got removed?

Reply:
cause we have 3 warnings for this in the doc.

Reply:
Realized that too after i finished reading 😅